src/Components/Modal/Modal.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './Modal.scss';
+
+class Modal extends React.Component {
+  render() {
+    const { open, close, header } = this.props;
+
+    return (
+      <>
+        <div className={open ? 'modal openModal' : 'modal'}>
```

Comment:
```suggestion
        <div className={`modal ${open ? openModal : ""}`}>
```
템플릿 리터럴을 사용하여 고정된 modal은 한번만 쓰고 변경되는 openModal의 형태만 관리할 수도 있을 것 같습니다!

Reply:
알려주신 방법으로 변경하였습니다. 클래스 관리와 가독성이 좋아지는 것 같습니다!